acme/acme/client.py

```diff
@@ -558,6 +558,62 @@ def new_account(self, new_account):
         return self._regr_from_response(response)
 
 
+class BackwardsCompatibleClientV2(object):
+    """ACME client wrapper that tends towards V2-style calls, but
+       supports V1 servers.
+
+       :ivar int acme_version:
```

Comment:
nit: You should write a small description here. Probably pointing out it's either 1 or 2 and these numbers correspond to the LE endpoint. (In terms of the ACME spec, v1 and v2 aren't accurate. We're on version 9 of the unfinished spec now.)